source/Applications/Basic/Visualization/CaptureHDRVisNormals/CaptureHDRVisNormals.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+This example shows how to capture Zivid point clouds, with color and normals.
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <pcl/visualization/cloud_viewer.h>
+
+#include <iostream>
+#include <thread>
+
+using namespace std::chrono_literals;
+
+namespace
+{
+    pcl::visualization::PCLVisualizer::Ptr normalsVisualizer(pcl::PointCloud<pcl::PointXYZRGB>::ConstPtr cloud,
+                                                             pcl::PointCloud<pcl::PointXYZRGBNormal>::ConstPtr normals)
+    {
+        // --------------------------------------------------------
+        // -----Open 3D viewer and add point cloud and normals-----
+        // --------------------------------------------------------
+        auto viewer = boost::make_shared<pcl::visualization::PCLVisualizer>("Viewer");
+
+        int view_rgb(0);
+        viewer->createViewPort(0.0, 0.0, 0.5, 1.0, view_rgb);
+        viewer->addText("Cloud RGB", 0, 0, "RGBText", view_rgb);
+        viewer->addPointCloud<pcl::PointXYZRGB>(cloud, "cloud", view_rgb);
+
+        int view_normals(0);
+        viewer->createViewPort(0.5, 0.0, 1.0, 1.0, view_normals);
+        viewer->addText("Cloud Normals", 0, 0, "NormalsText", view_normals);
+        viewer->addPointCloud<pcl::PointXYZRGBNormal>(normals, "cloudNormals", view_normals);
+        viewer->addPointCloudNormals<pcl::PointXYZRGBNormal, pcl::PointXYZRGBNormal>(normals,
+                                                                                     normals,
+                                                                                     10,
+                                                                                     1,
+                                                                                     "normals",
+                                                                                     view_normals);
+
+        viewer->setCameraPosition(0, 0, -100, 0, -1, 0);
+        return viewer;
+    }
+} // namespace
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        std::cout << "Configuring settings" << std::endl;
+        Zivid::Settings settings;
+        for(const auto aperture : { 9.57, 4.76, 2.59 })
+        {
+            std::cout << "Adding acquisition with aperture = " << aperture << std::endl;
+            const auto acquisitionSettings = Zivid::Settings::Acquisition{
+                Zivid::Settings::Acquisition::Aperture{ aperture },
+            };
+            settings.acquisitions().emplaceBack(acquisitionSettings);
+        }
+
+        std::cout << "Capturing frame (HDR)" << std::endl;
+        const auto frame = camera.capture(settings);
+        const auto pointCloud = frame.pointCloud();
+        const auto points = pointCloud.copyData<Zivid::PointXYZColorRGBA>();
```

Comment:
we call this "data" instead of "points" in other samples I think. We use "points" when using Zivid::PointXYZ only.